packages/ui/components/ui/box.tsx

```diff
@@ -28,11 +30,44 @@ export const Box = ({
   label,
   spacing,
   state,
-}: PropsWithChildren<VariantProps<typeof variants> & { label?: string }>) => {
+  layout,
+  layoutId,
+  headerContent,
+}: PropsWithChildren<
+  VariantProps<typeof variants> & {
+    label?: ReactNode;
+    layout?: boolean;
+    layoutId?: string;
```

Comment:
Do we need both of these? Couldn't we treat the presence of passing `layoutId` the equivalent as what just `layout` is being used for?

Also, is the naming clear enough to others to know this specifically relates to framer-motion? Think we should consider changing name to `framerLayout` or adding a code comment.

Reply:
In general, what does adding the layout field do?